src/libraries/adaptive-curve/ConstantsLib.sol

```diff
@@ -14,12 +16,12 @@ library ConstantsLib {
     /// @notice Target utilization (scaled by WAD).
     int256 public constant TARGET_UTILIZATION = 0.9 ether;
 
-    /// @notice Initial rate at target per second (scaled by WAD).
-    int256 public constant INITIAL_RATE_AT_TARGET = int256(0.04 ether) / 365 days;
+    /// @notice Initial rate at target per second (scaled by WAD) (4% APR <=> 1% min APR).
+    int256 public constant INITIAL_RATE_AT_TARGET = int256(0.01 ether) * CURVE_STEEPNESS / WAD_INT / 365 days;
```

Comment:
should we save `0.01 ether` as a constant like `ONE_BPS = 0.01 ether`?

Reply:
it's 100 bps though 😁

Reply:
Even better, let's call it one percent